pkgs/tools/misc/nvfancontrol/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub, libXNVCtrl, libX11, libXext }:
```

Comment:
```suggestion
{ lib, rustPlatform, fetchFromGitHub, libXNVCtrl, libX11, libXext }:
```
i don't think it's needed

Reply:
Yeah, forgot to remove it. Fixed in 045adfc.